src/components/Card/Card.tsx

```diff
@@ -14,14 +15,26 @@ export interface CardProps {
   ratio?: CardRatioOptions;
   flippedRatio?: boolean;
   invertInfoPosition?: boolean;
+  stacked?: boolean;
```

Comment:
let's add comments on top of the functions.
```js
/** puts the media slot on top of the info slot. disables the `ratio` behavior */
stacked?: boolean;
```

Every prop documentation here is important, but `mediaAspectRatio` has also special concerns which worth writing about

Reply:
cool. please add the rest of the comments later